src/identity/identity.ts

```diff
@@ -66,6 +66,7 @@ export interface IdentityOpts<optsT> {
  * Abstract class for Identity.
  */
 export class Identity {
+  chain: NetworkString;
```

Comment:
what's this needed for?

Reply:
couldn't it be inferred from `network`?

Reply:
NetworkString is the variable name of `network` so hard and lot of complexity, I think it's ok to store a simple string. An alternative could be the opposite: infer `network` from `chain` using `getNetwork` function, what's your opinion @altafan ?

Reply:
I think I solved this in liquidjs-lib psetv2 PR by [adding a filed](https://github.com/altafan/liquidjs-lib/blob/psetv2/ts_src/networks.ts#L22-L28) `name` to the `Newtork` type. The PR is ready to be merged and soon we will be able get rid of this `chain: NetworkString` param, do you agree?